### PR TITLE
[Console][WebServerBundle] Use "exec" when possible

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 3.3
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #23568
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Symfony/Bundle/WebServerBundle/composer.json [hunk 1]
```diff
@@ -17,9 +17,9 @@
     ],
     "require": {
         "php": ">=5.5.9",
-        "symfony/console": "~2.8.8|~3.0.8|~3.1.2|~3.2",
+        "symfony/console": "~3.3",
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/WebServerBundle/composer.json]: there is no need to keep pre-3.3 deps for 3.3-introduced bundles IMHO, that makes things easier

**Comment by robfrawley** [comment]
Confirmed to fix the issue described in #23568 for me.

File: src/Symfony/Bundle/WebServerBundle/WebServer.php [hunk 2]
```diff
@@ -151,11 +150,11 @@ private function createServerProcess(WebServerConfig $config)
             throw new \RuntimeException('Unable to find the PHP binary.');
         }
 
-        $builder = new ProcessBuilder(array($binary, '-S', $config->getAddress(), $config->getRouter()));
-        $builder->setWorkingDirectory($config->getDocumentRoot());
-        $builder->setTimeout(null);
+        $process = new Process(array($binary, '-S', $config->getAddress(), $config->getRouter()));
+        $process->setWorkingDirectory($config->getDocumentRoot());
+        $process->setTimeout(null);
```
Comment (chalasr) [inline src/Symfony/Bundle/WebServerBundle/WebServer.php]: `$process = new Process(array($binary, '-S', $config->getAddress(), $config->getRouter()), $config->getDocumentRoot(), null, null);` btw?

**Review by chalasr — APPROVED** [review]

File: src/Symfony/Bundle/WebServerBundle/WebServer.php (hunk 2 above)
Comment (nicolas-grekas) [inline src/Symfony/Bundle/WebServerBundle/WebServer.php]: More readable as is to me.

Comment (chalasr) [inline src/Symfony/Bundle/WebServerBundle/WebServer.php]: right!

**Review by ogizanagi — APPROVED** [review]

**Review by xabbuh — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**